Cargo.toml

```diff
@@ -22,6 +22,7 @@ nb = "0.1.2"
 stm32f7 = "0.11.0"
 micromath = "1.0.0"
 synopsys-usb-otg = { version = "0.2.0", features = ["cortex-m"], optional = true }
+stm32-fmc = { version = "0.1.2", features = ["sdram"], optional = true }
```

Comment:
I get a compiler error when attempting to use version 0.2.0:

```
error[E0107]: wrong number of type arguments: expected 2, found 1
  --> /Users/ben/Documents/projects/stm32f7xx-hal/src/fmc.rs:75:39
   |
75 |     fn sdram<BANK: SdramPinSet, PINS: PinsSdram<BANK>, CHIP: SdramChip>(
   |                                       ^^^^^^^^^^^^^^^ expected 2 type arguments

error: aborting due to previous error
```

Might be good to have this crate work with the latest  version prior to merging.